docs/version0.38.md

```diff
@@ -28,6 +28,7 @@ The following new features and notable changes since version 0.37.0 are included
 
 - [New binaries and changes to supported environments](#binaries-and-supported-environments)
 - [New `-XX:[+|-]HandleSIGUSR2` option added](#new-xx-handlesigusr2-option-added)
+- [Technical preview of CRIU feature](#technical-preview-of-criu-feature)
```

Comment:
I would say "CRIU support" here. Yes it's a feature of OpenJ9, but that feature is to support the use of CRIU, which is a separate project in its own right.

Reply:
Done